Sources/SnowManLogic/ViewLogic/View.swift

```diff
@@ -0,0 +1,44 @@
+public class View: Viewable {
+	var ui: Receivable 
+	var guess: String!
+
+	public init(ui: Receivable) {
+		self.ui = ui
+	}
+
+	public func receiveGuess() -> String {
```

Comment:
the logic here is like, basically the same as in the mock, the mock/stub object should be void of logic.